popmon/visualization/histogram_section.py

```diff
@@ -189,3 +205,96 @@ def _plot_histograms(feature, date, hc_list, hist_names):
         plot = ""
 
     return {"name": date, "description": get_stat_description(date), "plot": plot}
+
+
+def _plot_heatmap(feature, date, hc_list) :
+    
+    hist_names = [' Heatmap ', ' Column Normalized Heatmap ',  ' Row Normalized Heatmap ' ]
+    top_lim = 20
```

Comment:
Users might want to change this parameter. Suggestion:
pass it to _plot_heatmap and set it in HistogramSection similar to `last_n`